source/development/rbac-database-integrity.rst

```diff
@@ -1,18 +1,16 @@
 .. Copyright (C) 2022 Wazuh, Inc.
 
-.. _rbac_database_integrity:
-
 RBAC database integrity
 =======================
 
-The RBAC database integrity is checked every time the Wazuh API starts to determine if the database should be updated. The integrity check allows us to cover the following cases:
+The integrity of the RBAC database is checked before determining if the database should be updated. The integrity check allows us the following:
```

Comment:
With this change, we don't indicate that the integrity check is done when the API starts. Maybe the best sentence is the following, or something similar to the following:
```suggestion
The integrity of the RBAC database is checked when the API starts. The result of this check determines whether the database needs an update or not. The integrity check allows us the following:
```